barman/output.py

```diff
@@ -709,17 +709,50 @@ def render_show_backup_general(backup_info, output_fun, row):
         :param str row: format string which allows for `key: value` rows to be
             formatted
         """
-        if "backup_name" in backup_info and backup_info["backup_name"] is not None:
-            output_fun(row.format("Backup Name", backup_info["backup_name"]))
+        backup_name = backup_info.get("backup_name")
+        if backup_name:
+            output_fun(row.format("Backup Name", backup_name))
+
         output_fun(row.format("Server Name", backup_info["server_name"]))
-        if backup_info["systemid"]:
+
+        backup_name = backup_info.get("systemid")
+        if backup_name:
             output_fun(row.format("System Id", backup_info["systemid"]))
+
         output_fun(row.format("Status", backup_info["status"]))
         if backup_info["status"] in BackupInfo.STATUS_COPY_DONE:
             output_fun(row.format("PostgreSQL Version", backup_info["version"]))
             output_fun(row.format("PGDATA directory", backup_info["pgdata"]))
-            output_fun(row.format("Checksums", backup_info["data_checksums"]))
-            output_fun("")
+            cluster_size = backup_info.get("cluster_size")
+            if cluster_size:
+                output_fun(
+                    row.format("Estimated Cluster Size", pretty_size(cluster_size))
+                )
+                output_fun("")
```

Comment:
```suggestion
            output_fun("")
```

This should be out of the `if` it seems.